presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -511,6 +512,19 @@ public HiveClientConfig setRespectTableFormat(boolean respectTableFormat)
         return this;
     }
 
+    public boolean isInsertOverwriteImmutablePartitionEnabled()
+    {
+        return insertOverwriteImmutablePartitions;
+    }
+
+    @Config("hive.insert-overwrite-immutable-partitions-enabled")
+    @ConfigDescription("Can insertion query overwrites existing partitions when partitions are immutable. This config only takes effect with hive.immutable-partitions set to true")
```

Comment:
nit: how about saying `When enabled, insertion query can overwrite existing partitions when partitions are immutable. This...`